packages/earth-admin/src/pages-client/dashboards/details.tsx

```diff
@@ -0,0 +1,307 @@
+/*
+  Copyright 2018-2020 National Geographic Society
+  Use of this software does not constitute endorsement by National Geographic
+  Society (NGS). The NGS name and NGS logo may not be used for any purpose without
+  written permission from NGS.
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+  this file except in compliance with the License. You may obtain a copy of the
+  License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed
+  under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+  CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations under the License.
+*/
+
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import renderHTML from 'react-render-html';
+import { Controller, useForm } from 'react-hook-form';
+
+import {
+  encodeQueryToURL,
+  formatDate,
+  flattenArrayForSelect,
+} from 'utils';
+import { useRequest } from 'utils/hooks';
+
+import { noSpecialCharsRule, alphaNumericDashesRule, setupErrors } from 'utils/validations';
+
+import { AsyncSelect } from '@marapp/earth-components';
+import { LinkWithOrg } from 'components/link-with-org';
+import { ActionModal } from 'components/action-modal';
+import { Input } from 'components/input';
+import { InlineEditCard } from 'components/inline-edit-card';
+import { Card } from 'components/card';
+import { Toggle } from 'components/toggle';
+import { HtmlEditor } from 'components/html-editor';
+import { DetailList } from 'components/detail-list';
+import { ErrorMessages } from 'components/error-messages';
+
+import { useAuth0 } from 'auth/auth0';
+import { AuthzGuards } from 'auth/permissions';
+import { ContentLayout } from 'layouts';
+
+
+import { CUSTOM_STYLES, SELECT_THEME } from '../../theme';
+import { getAllWidgets, getDashboard, handleDashboardForm } from 'services';
+
+const DASHBOARD_DETAIL_QUERY = {
+  include: 'layers,widgets',
+  select: 'layers.id,layers.name,layers.type,widgets.id,widgets.name,widgets.type',
+  sort: 'layers.name,widgets.name',
+};
+
+export function DashboardDetail(path: any) {
+  const {getPermissions, selectedGroup} = useAuth0();
+  const writePermissions = getPermissions(AuthzGuards.writeDashboardsGuard);
+
+  const encodedQuery = encodeQueryToURL(`dashboards/${path.page}`, {
+    ...DASHBOARD_DETAIL_QUERY,
+    ...{group: selectedGroup},
+  });
+
+  const {isLoading, data} = useRequest(() => getDashboard(encodedQuery), {
+    permissions: AuthzGuards.writeDashboardsGuard,
```

Comment:
useRequest no longer handles authorization and relies on the server to do it. We no longer need to pass it the permissions field